src/main/scala/io/lenses/data/generator/http/LensesClient.scala

```diff
@@ -0,0 +1,89 @@
+package io.lenses.data.generator.http
+
+import cats.effect.IO
+import org.http4s.Uri
+import org.http4s.client.Client
+import org.http4s.Request
+import org.http4s.circe._
+import io.circe.Json
+import org.http4s.Method
+import org.http4s.Headers
+import org.http4s.Header
+import org.apache.avro
+import org.apache.avro.Schema
+import io.lenses.data.generator.cli.LensesCreds
+
+object LensesClient {
+  final case class AuthToken(value: String) extends AnyVal
+
+  def apply(
+      baseUrl: Uri,
+      client: Client[IO]
+  ): LensesClient =
+    new LensesClient {
+
+      override def setTopicMetadata(topicName: String, schema: Schema)(implicit
+          auth: AuthToken
+      ) = {
+        val request =
+          Request[IO](
+            method = Method.POST,
+            uri = baseUrl / "api" / "v1" / "metadata" / "topics"
+          )
+            .withHeaders(
+              Header("X-Kafka-Lenses-Token", auth.value)
+            )
+            .withEntity(
+              Json.obj(
+                "topicName" -> Json.fromString(topicName),
+                "keyType" -> Json.fromString("BYTES"),
```

Comment:
Why Bytes?
We should generate topics with Key being populated

Reply:
Fair, this is a first pass, but I agree we should also generate a schema for the key